src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -120,8 +120,8 @@
 
   public static final String TIMESTAMP_COLUMN_NAME_CONFIG = "timestamp.column.name";
   private static final String TIMESTAMP_COLUMN_NAME_DOC =
-      "The name of the timestamp column to use to detect new or modified rows. This column may "
-      + "not be nullable.";
+      "Comma separated list of one or more timestamp columns to detect new or modified rows in "
+      + "the order specified. At least one column may not be nullable.";
```

Comment:
To address @wicknicks' suggestion, we might change this to something like:

> Comma separated list of one or more timestamp columns to detect new or modified rows. The order is important: the first non-null timestamp found in each row will be used. At least one column should not be nullable.

Reply:
Or

> Comma separated list of one or more timestamp columns to detect new or modified rows using the `COALESCE` SQL function. Rows whose first non-null timestamp value is greater than the largest previous timestamp value seen will be discovered with each poll. At least one column should not be nullable.

Reply:
Looks like we are assuming that there exists an order to how the timestamps are updated. Could there be cases where this is not true? For example, in an `employee` table we could have columns like `created_date`, `name_update_date` and `address_update_date`, then we might not be able to prescribe an order suitable to the `COALESCE` function.

Reply:
What would a query against that table look like, especially the ordering clause?